statsmodels/tools/tests/test_linalg.py

```diff
@@ -21,3 +22,25 @@ def test_stationary_solve_2d():
     soln = np.linalg.solve(tmat, b)
     soln1 = linalg.stationary_solve(r, b)
     assert_allclose(soln, soln1, rtol=1e-5, atol=1e-5)
+
+
+def test_pinv_equivalence():
+    # This test was moved from the __main__ section of tools.linalg
+    # Note on Windows32:
+    #    linalg doesn't always produce the same results in each call
+    import scipy.linalg
+    a0 = np.random.randn(100, 10)
+    b0 = a0.sum(1)[:, None] + np.random.randn(100, 3)
+
+    # smoke-tests for lstsq, pinv, pinv2
+    linalg.lstsq(a0, b0)
+    linalg.pinv(a0)
+    linalg.pinv2(a0)
```

Comment:
pinv and pinv2 are called again below, so there is no need to smoke test them

lstsq could get the same assert against scipy.linalg.lstsq as the other two

Reply:
Sounds good, will update.